### ISSUE TITLE
Found the following (possibly) invalid URLs

### ISSUE THREAD

**Issue by VincentGuyader** [ThinkR-open/golem#980]
## User - Validation 

+ [ ] url check are OK

## Dev - Tech
```
Found the following (possibly) invalid URLs:
  URL: https://r-pkgs.org/Metadata.html#sec-description
    From: inst/doc/a_start.html
    Status: 404
    Message: Not Found
  URL: https://r-pkgs.org/Metadata.html#sec-namespace
    From: inst/doc/b_dev.html
    Status: 404
    Message: Not Found
  URL: https://r-pkgs.org/tests.html
    From: inst/doc/a_start.html
    Status: 404
    Message: Not Found
```

**Comment by ColinFay** [comment]
Current output : 

```r
> urlchecker::url_check()
✖ Error: man/get_sysreqs.Rd:21:36 Error: SSL certificate problem: self signed certificate
system requirements using the \url{https://sysreqs.r-hub.io}
                                   ^~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by ColinFay** [comment]
closed via 4b05eb8
